README.md

```diff
@@ -1 +1,13 @@
-# iabtcf-java
\ No newline at end of file
+# iabtcf-java
+
+## Using It
+
+### Maven
+
+```
+<dependency>
+  <groupId>com.iabtcf</groupId>
+  <artifactId>iabtcf-core</artifactId>
+  <version>1.0.2-alpha.0</version>
```

Comment:
isn't it easier to point to maven repo url so that user can get the latest version?

Reply:
ya good point